Makefile

```diff
@@ -47,23 +61,69 @@ vet:
 	@echo ">> vetting code"
 	@$(GO) vet $(pkgs)
 
-build: init
-	@echo ">> building binaries"
-	@$(PROMU) build --prefix $(PREFIX)
+# It's just alias to build binary
+build: release
+
+snapshot: $(GOPATH)/bin/goreleaser
+	@echo ">> building snapshot"
+	goreleaser --snapshot --skip-sign --skip-validate --skip-publish --rm-dist
 
-tarball: init
-	@echo ">> building release tarball"
-	@$(PROMU) tarball --prefix $(PREFIX) $(BIN_DIR)
+# We use this target name to build binary across all PMM components
+release:
+	@echo ">> building binary"
+	@CGO_ENABLED=0 $(GO) build -v -a \
+		-tags 'netgo' \
+		-ldflags '\
+		-X '$(GO_PACKAGE)/vendor/github.com/percona/pmm/version.ProjectName=$(BIN_NAME)' \
+		-X '$(GO_PACKAGE)/vendor/github.com/percona/pmm/version.Version=$(APP_VERSION)' \
+		-X '$(GO_PACKAGE)/vendor/github.com/percona/pmm/version.PMMVersion=$(PMM_RELEASE_VERSION)' \
+		-X '$(GO_PACKAGE)/vendor/github.com/percona/pmm/version.Timestamp=$(PMM_RELEASE_TIMESTAMP)' \
+		-X '$(GO_PACKAGE)/vendor/github.com/percona/pmm/version.FullCommit=$(PMM_RELEASE_FULLCOMMIT)' \
+		-X '$(GO_PACKAGE)/vendor/github.com/percona/pmm/version.Branch=$(PMM_RELEASE_BRANCH)' \
+		-X '$(GO_PACKAGE)/vendor/github.com/prometheus/common/version.BuildUser=$(USER)@$(TRAVIS_APP_HOST)' \
+		'\
+		-o $(BIN_DIR)/$(BIN_NAME) .
+
+community-release: $(GOPATH)/bin/goreleaser
+	@echo ">> building release"
+	goreleaser release --rm-dist --skip-validate
 
 docker:
 	@echo ">> building docker image"
 	@docker build -t "$(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG)" .
 
-init:
+$(GOPATH)/bin/dep:
+	curl -s https://raw.githubusercontent.com/golang/dep/v0.5.0/install.sh | sh
+
+$(GOPATH)/bin/gocoverutil:
```

Comment:
We don't actually need `gocoverutil` with Go 1.11+: it supports `go test -coverprofile=coverage.txt ./...` (`coverprofile` with several packages) natively.